Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -110,12 +110,22 @@ extension SettingsDictionary {
 
     /// Sets `"OTHER_SWIFT_FLAGS"` to `flags`
     public func otherSwiftFlags(_ flags: String...) -> SettingsDictionary {
-        merging(["OTHER_SWIFT_FLAGS": SettingValue(flags.joined(separator: " "))])
+        otherSwiftFlags(flags)
+    }
+
+    /// Sets `"OTHER_SWIFT_FLAGS"` to `flags`
+    public func otherSwiftFlags(_ flags: [String]) -> SettingsDictionary {
```

Comment:
nit: should we add an inherited boolean as well? I think many time you forget to pass it and it causes problems.
Adding a parameter forces the user to think about it.

Reply:
Which boolean are you talking about here ?

Reply:
literalli an `inherited: Bool` that adds `$(inherited)` if true

Reply:
But let's see what @pepicrft @fortmarek @kwridan @waltflanagan  think about it

Reply:
I'm fine with either option, but I'd probably lean to not including it to keep the API surface of this method smaller. If we decide to go ahead with it, it should probably be added to most of these helper methods, not just this one